### PR TITLE
FER template description updated

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add the missing description for the FER templates.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Update the FER templates with missing info](https://sumologic.atlassian.net/browse/DOCS-422)

### CODE REVIEW

File: docs/manage/field-extractions/fer-templates/apache-access-logs.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ parse regex "^(?<src_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"
...
-| status_code | Status code that the server sends back to the client | 200 |
-| size | Size of the object returned to the client | 8825 |
-| referrer | Site that the client reports having been referred from | [http://www.google.com/url?sa=t&rct=j...source=web&cd=4](http://www.google.com/url?sa=t&rct=j&q=log-reduce&source=web&cd=4) |
-| user_agent  | Identifying information that the client browser reports about itself | Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_7; en-us) AppleWebKit/533.21.1 (KHTML, like Gecko) Chrome/19.0.1084.30 Safari/536.5 |
+| src_ip | IP address of the client (remote host) from which the request is made to the server. | 221.125.19.252 |
+| method | Method used by the client. | GET |
+| url | Resource requested by the client. | `v` |
+| status_code | Status code that the server sends back to the client. | 200 |
+| size | Size of the object returned to the client. | 8825 |
+| referrer | Displays the site that the client reports having been referred from. | [http://www.google.com/url?sa=t&rct=j...source=web&cd=4](http://www.google.com/url?sa=t&rct=j&q=log-reduce&source=web&cd=4) |
```
Comment (kimsauce) [inline docs/manage/field-extractions/fer-templates/apache-access-logs.md:34]:
```suggestion
| referrer | Displays the site that the client reports having been referred from. | `http://www.google.com/url?sa=t&rct=j...source=web&cd=4` |
```

File: docs/manage/field-extractions/fer-templates/apache-access-logs.md [hunk 2]
```diff
@@ -26,10 +26,10 @@ parse regex "^(?<src_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"
...
-| src_ip | IP address of the client (remote host) which made the request to the server | 221.125.19.252 |
-| method | Method used by the client | GET |
-| url | Resource requested by the client | `v` |
-| status_code | Status code that the server sends back to the client | 200 |
-| size | Size of the object returned to the client | 8825 |
-| referrer | Site that the client reports having been referred from | [http://www.google.com/url?sa=t&rct=j...source=web&cd=4](http://www.google.com/url?sa=t&rct=j&q=log-reduce&source=web&cd=4) |
-| user_agent  | Identifying information that the client browser reports about itself | Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_7; en-us) AppleWebKit/533.21.1 (KHTML, like Gecko) Chrome/19.0.1084.30 Safari/536.5 |
+| src_ip | IP address of the client (remote host) from which the request is made to the server. | 221.125.19.252 |
+| method | Method used by the client. | GET |
+| url | Resource requested by the client. | `v` |
```
Comment (kimsauce) [inline docs/manage/field-extractions/fer-templates/apache-access-logs.md:31]: should `v` be a url or is that intentional?

File: docs/manage/field-extractions/fer-templates/nginx-logs.md [hunk 3]
```diff
@@ -27,10 +27,10 @@ parse regex "^(?<src_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"
...
-| status_code | HTTP response code from server | 304 |
-|  size | Size of server response in bytes | 0 |
-|  referrer | Referral URL | [https://www.sumologic.com/aws/elb/aws-elastic-load-balancers-classic-vs-application/](https://www.sumologic.com/aws/elb/aws-elastic-load-balancers-classic-vs-application/) |
-|  user_agent | Information about the client browser | Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:54.0) Gecko/20100101 Firefox/54.0 |
+| src_ip | The source IP address. | `205.197.2.175` |
+| method | HTTP request method. | GET |
+| url | Resource requested by the client. | `/wp-content/uploads/Screen-Shot-2017-04-13-at-7.12.35-PM-231x300.png` |
+| status_code | HTTP response status code. | 304 |
+|  size | The size of the object returned to the client. | 0 |
+|  referrer | The website from which the client reports are referred. | [AWS Elastic Load Balancer](https://www.sumologic.com/aws/elb/aws-elastic-load-balancers-classic-vs-application/) |
```
Comment (kimsauce) [inline docs/manage/field-extractions/fer-templates/nginx-logs.md:35]: should the referrer be--

```suggestion
|  referrer | The website from which the client reports are referred. | _https://www.sumologic.com/aws/elb/aws-elastic-load-balancers-classic-vs-application_ |
```

File: docs/manage/field-extractions/fer-templates/varnish-logs.md [hunk 4]
```diff
@@ -26,10 +26,10 @@ parse regex "^(?<src_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"
...
-| status_code | The status code that the server sends back to the client | 304 |
-| size | The size of the object returned to the client | 5201 |
-| referrer | The site that the client reports having been referred from | [http://search.yahoo.com/mobile/s?rew...0logs&pintl=en](http://search.yahoo.com/mobile/s?rewrite=72&.tsrc=log&first=1&p=AWS-logs&pintl=en) |
-| user_agent  | The identifying information that the client browser reports about itself | Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:23.0) Gecko/20131011 Firefox/23.0 |
+| src_ip | The source IP address.  | `101.92.120.16` |
+| method | HTTP request method. | GET |
+| url | Resource requested by the client.  | `/_includes/wp/blog/wp-content/plugins/us/31063765-bpfull.phpi?&w=50&id=6&random=1331063765` |
+| status_code | HTTP response status code. | 304 |
+| size | The size of the object returned to the client. | 5201 |
+| referrer | The website from which the client reports are referred. | [Yaho.com](http://search.yahoo.com/mobile/s?rewrite=72&.tsrc=log&first=1&p=AWS-logs&pintl=en) |
```
Comment (kimsauce) [inline docs/manage/field-extractions/fer-templates/varnish-logs.md:34]: should this be..

```suggestion
| referrer | The website from which the client reports are referred. | `http://search.yahoo.com/mobile/s?rewrite=72&.tsrc=log&first=1&p=AWS-logs&pintl=en` |
```

**Review by kimsauce — APPROVED** [review]
had a few questions!

File: docs/manage/field-extractions/fer-templates/apache-access-logs.md (hunk 2 above)
Comment (JV0812) [inline docs/manage/field-extractions/fer-templates/apache-access-logs.md:31]:
```suggestion
| url | Resource requested by the client. | `/_js/master.js` |
```

Comment (JV0812) [reply docs/manage/field-extractions/fer-templates/apache-access-logs.md]: Fixed the missing path.

**Comment by jpipkin1** [comment]
@JV0812, does this PR need a do-not-merge label?

**Pull request merged**